[2020-11-25 17:03:02,034: INFO/MainProcess] Connected to redis://localhost:6379//
[2020-11-25 17:03:03,909: INFO/MainProcess] mingle: searching for neighbors
[2020-11-25 17:03:08,941: INFO/MainProcess] mingle: all alone
[2020-11-25 17:03:10,950: WARNING/MainProcess] /usr/local/lib/python3.7/dist-packages/celery/fixups/django.py:203: UserWarning: Using settings.DEBUG leads to a memory
            leak, never use this setting in production environments!
  leak, never use this setting in production environments!''')
[2020-11-25 17:03:10,952: INFO/MainProcess] celery@worker1 ready.
